### ISSUE TITLE
Automatically walking in town doesn't work

### ISSUE THREAD

**Issue by Fjurg** [hectorgimenez/koolo#350]
I've set a cycle of three areas (meph, andy and pit). 

After starting in Rogue Encampment I manually have to click and hold the mouse-button to make the character walk (it will walk according to the bots' rules). When I then use WP to eg. Outer Cloister then it starts moving and doing awesome things. I believe my configuration to be according to the specification in the README. 

The latest log statement is:
`time=11:03:03 level=INFO msg="Visiting vendor..." forceRefill=false`

tl;dr: My character moves in hostile areas but not in towns.

**Comment by Fjurg** [comment]
Ok so I am pretty stupid. This is due to the fact that the bot is using 'e' as the forceMove. As I had bound it to a skill it was unable to move whatsoever and was only teleporting around. Closing this.
